### ISSUE TITLE
Is this project deprecated ?

### ISSUE THREAD

**Issue by tobecrazy** [ruby-china/ruby-china-android#4]
I'm an Android developer, I saw ruby china already release both Android and iOS client. But it seems Android project is out of date. So we deprecated this project now?

**Comment by chloerei** [comment]
Yes, it's no longer maintained. Web already suffices for most needs.
